### PR TITLE
molecule created for downtime page

### PR DESCRIPTION
**Pull request by riya-2206**
molecule created for downtime page 
Screenshot 2024-03-06 135802

### CODE REVIEW

File: src/molecules/DowntimePage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      <Box className={styles.container}>
+        <Box className={styles.title} style={{color: config.theme.tertiaryColor.value}}>{config.component.title}</Box>
+        <Box className={styles.imageContainer}>
+        {/* Contains the down time gif in css */}
+        </Box>
+        <Box className={styles.supportingText} style={{color: config.theme.secondaryColor.value}}>{config.component.supportingText}</Box>
+        <Box className={styles.dialerBox}>
+          <Button variant={"text"} style={{gap:"12px", textTransform: 'none'}} onClick={()=>console.log(config.component.contactLink)}
+          // href={`tel:${flags.dialer_number.value}`}
+            >
```
Comment (geeky-abhishek) [inline src/molecules/DowntimePage/index.tsx]: read the image link from config and also pass a fallback icon related to general downtime

File: src/molecules/DowntimePage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        </Box>
+
+        <Box display={"flex"} justifyContent={"space-around"} mx={30}>
+          <Button
+            className={styles.retryButton}
+            onClick={() => window?.location.reload()}
+            style={{ textTransform: 'none', backgroundColor: config.theme.secondaryColor.value, color:"white", padding: "1vh 2vh" }}
+            id='reloadButton'
+            >
+            {config.component.refreshTextButton}
```
Comment (geeky-abhishek) [inline src/molecules/DowntimePage/index.tsx]: use proper key like confif.component.refreshText also always have a fallback text

File: src/molecules/DowntimePage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          </Button>
+           <Button
+          className={styles.viewPrevChatsButton}
+          style={{ textTransform: 'none', backgroundColor: config.theme.tertiaryColor.value, color:"white", padding: "1vh 2vh" }}
+          onClick={() => {
+            console.log(config.component.previousPageButton)
```
Comment (geeky-abhishek) [inline src/molecules/DowntimePage/index.tsx]: define the callback above in the code and pass it here

File: src/molecules/DowntimePage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            className={styles.retryButton}
+            onClick={() => window?.location.reload()}
+            style={{ textTransform: 'none', backgroundColor: config.theme.secondaryColor.value, color:"white", padding: "1vh 2vh" }}
+            id='reloadButton'
+            >
+            {config.component.refreshTextButton}
+          </Button>
+           <Button
+          className={styles.viewPrevChatsButton}
+          style={{ textTransform: 'none', backgroundColor: config.theme.tertiaryColor.value, color:"white", padding: "1vh 2vh" }}
```
Comment (geeky-abhishek) [inline src/molecules/DowntimePage/index.tsx]: try to use only primary or secondary color

File: src/molecules/DowntimePage/index.tsx (hunk 1 above)
Comment (riya-2206) [reply src/molecules/DowntimePage/index.tsx]: done

File: src/molecules/DowntimePage/index.tsx (hunk 2 above)
Comment (riya-2206) [reply src/molecules/DowntimePage/index.tsx]: done

File: src/molecules/DowntimePage/index.tsx (hunk 3 above)
Comment (riya-2206) [reply src/molecules/DowntimePage/index.tsx]: done

File: src/molecules/DowntimePage/index.tsx (hunk 4 above)
Comment (riya-2206) [reply src/molecules/DowntimePage/index.tsx]: done

File: src/molecules/DowntimePage/index.test.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+import DowntimePage from './index';
+
```
Comment (geeky-abhishek) [inline src/molecules/DowntimePage/index.test.tsx]: @riya-2206 they are very basic test cases and are testing nothing from config.Plz import config and check if components are properly rendering based on the config.

File: src/molecules/DowntimePage/index.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styles from './index.module.css';
+import { Box, Button, Typography} from '@mui/material';
+import config from './config.json';
+
+const DowntimePage: React.FC = () => {
+  const handleRefreshClick = ()=>{
```
Comment (geeky-abhishek) [inline src/molecules/DowntimePage/index.tsx]: use useCallback for every function like constHandleRefreshClick =useCallback(()=>{},[])

**Comment by riya-2206** [comment]
Have merged with base-setup and used customThemeHook for changing theme
image

**Pull request merged**